tsconfig.json

```diff
@@ -6,15 +6,16 @@
     "downlevelIteration": true,
     "moduleResolution": "node",
     "esModuleInterop": true,
-    "lib":[ "es2017" ],
+    "lib":[ "es2017","dom" ],
     "declaration": true,
     "sourceMap": true,
     "plugins": [
       { "name": "typescript-tslint-plugin" }
     ]
   },
   "include": [
-    "lib/**/*.ts"
+    "lib/**/*.ts",
+    "./typings-custom/**/*.ts"
```

Comment:
I'm not able to find any "typings-custom" files.  Why is this needed?

Reply:
Sorry, I forgot to remove that, it's actually not necessary.